### DISCUSSION TITLE
Vercel says deployment successful, 404 error when visiting page.

### DISCUSSION THREAD

**Discussion by DannyBuglak** [vercel/community#5735] [Help]
### Summary

I am trying to deploy my webpage which I built using purely HTML, CSS, and JavaScript, no framework. I deployed with Vercel and I have a status that it is running and there are no logs available either. What could be the issue?

### Steps to Reproduce

404: NOT_FOUND
Code: NOT_FOUND
ID: cle1::7n7r6-1707087606919-a1ae3042c2f0

**Comment by DannyBuglak** [comment]
Resolved, the structure of my project was messed up.

**Comment by Dennis00000** [comment]
Hey did you get the solution to it?
